build_libopus.bat

```diff
@@ -25,6 +25,15 @@ IF ERRORLEVEL 1 (
 
 copy %PLATFORM%\%CONFIG%\opus.lib %LIB_DIR%
```

Comment:
how about replacing this with a copy of *.lib and *.pdb?

Reply:
Would be better indeed.